check_limits.py

```diff
@@ -1,18 +1,40 @@
+from bms import validate_bms_param_name
+from bms import validate_bms_param_value
+from bms import validate_overall_bms_health
+from bms import set_system_language
+from bms import set_sensor_temperatue_unit
 
-def battery_is_ok(temperature, soc, charge_rate):
-  if temperature < 0 or temperature > 45:
-    print('Temperature is out of range!')
-    return False
-  elif soc < 20 or soc > 80:
-    print('State of Charge is out of range!')
-    return False
-  elif charge_rate > 0.8:
-    print('Charge rate is out of range!')
-    return False
-
-  return True
+if __name__ == '__main__':
+    
+    assert (set_system_language('fr', mode=False)==False)
+    assert (set_system_language('de', mode=False)==True)
+    set_system_language()
```

Comment:
Any way to test the language?

Reply:
Yes, this method is already checking the same, hence i kept the assert statement for different language here, for example French is not supported in the designed module, hence it is returning false here,
Also for one case i haven't passed any parameter for English as it is set as default if you call this method
Below is the glance of the method i had defined already in bms.py file 

![image](https://user-images.githubusercontent.com/15873999/110743131-8d5a3200-825d-11eb-85e3-4d977672e277.png)